Libraries/Components/ViewPager/ViewPagerAndroid.android.js

```diff
@@ -144,6 +160,12 @@ var ViewPagerAndroid = React.createClass({
     }
   },
```

Comment:
Should I add `type Event = Object;` to fix the flow lint warning?
It seems there are some Event usage in original ViewPagerAndroid  such as `_onPageScroll` and `_onPageSelected`.